src/nvim/ui.c

```diff
@@ -18,6 +18,7 @@
 #include "nvim/ex_getln.h"
 #include "nvim/fold.h"
 #include "nvim/garray.h"
+#include "nvim/globals.h"
```

Comment:
I don't think it is necessary because it is included in the following section of `vim.h`.

https://github.com/neovim/neovim/blob/7b2b44bce4bff85d9fd1bef09d294ddb889056e4/src/nvim/vim.h#L296

Reply:
Thanks, I've removed this

Reply:
Just for reference: technically the "include what you use" goal (https://github.com/neovim/neovim/issues/549) dictates that we don't want to depend on transitive includes.